xbmc/music/GUIViewStateMusic.cpp

```diff
@@ -188,6 +189,8 @@ CGUIViewStateMusicDatabase::CGUIViewStateMusicDatabase(const CFileItemList& item
       AddSortMethod(SortByArtist, sortAttribute, 557, LABEL_MASKS("%F", "", strAlbumLeft, strAlbumRight));  // Filename, empty | Userdefined, Userdefined
       // year
       AddSortMethod(SortByYear, 562, LABEL_MASKS("%F", "", strAlbumLeft, strAlbumRight));
+      // album date added
+      AddSortMethod(SortByDateAdded, sortAttribute, 570, LABEL_MASKS("%F", "", strAlbumLeft + " - " + strAlbumRight, "%a"));  // Filename, empty | Userdefined - Userdefined, dateAdded
```

Comment:
Hrm, but this might also be a bad idea :/
http://kodi.wiki/view/advancedsettings.xml#musiclibrary
It's colliding a bit with that. We should have a way to show the value we're sorting by, but the user also wants to be able to format the strings by himself. And he probably thinks he is defining left | right. Not left - right.